commands/acr-build.ts

```diff
@@ -0,0 +1,99 @@
+import { ContainerRegistryManagementClient } from 'azure-arm-containerregistry';
+import { QuickBuildRequest } from "azure-arm-containerregistry/lib/models";
+import { Registry } from 'azure-arm-containerregistry/lib/models';
+import { ResourceManagementClient } from 'azure-arm-resource';
+import { BlobService, createBlobServiceWithSas } from "azure-storage";
+import * as vscode from "vscode";
+import { ImageNode } from "../explorer/models/imageNode";
+import { AzureUtilityManager } from "../utils/azureUtilityManager";
+import { acquireResourceGroup, acquireSubscription, quickPickACRRegistry } from './utils/quick-pick-azure';
+let tar = require('tar');
+let fs = require('fs');
+let os = require('os');
+let url = require('url');
+
+export async function queueBuild(context?: ImageNode): Promise<void> {
+
+    console.log("Obtaining Subscription and Client");
```

Comment:
>console [](start = 4, length = 7)

Is it a common pattern to use console.log for logging in current extension?

Reply:
No, planning on working Esteban on how his logs can be used for build. This was just in mostly for working with it until we push to the master docker extension.